src/libraries/System.Net.Http/tests/StressTests/HttpStress/build-local.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+## This is a helper script for non-containerized local build and test execution.
+## It downloads and uses the daily SDK which contains the compatible AspNetCore bits.
+## Usage:
+## ./build-local.sh [StressConfiguration] [LibrariesConfiguration]
+
+version=7.0
+repo_root=$(git rev-parse --show-toplevel)
+daily_dotnet_root=./.dotnet-daily
+
+stress_configuration="Release"
+if [ "$1" != "" ]
+then
+    stress_configuration=${1,,}                   # Lowercase all characters in $1
+    stress_configuration=${stress_configuration^} # Uppercase first character
+fi
+
+libraries_configuration="Release"
```

Comment:
Maybe Debug might be better default option??? I assume that most of us build libs in Debug (only CLR in Release).

Reply:
I always build libraries in Release when it comes to stress testing, but maybe it's just me. Doesn't a Debug build change timing characteristics significantly, meaning that you have less chance to reproduce an issue that has been detected in Release?

Reply:
Preferably, one could choose. While Release is what we ship, Debug builds have more Asserts and extra checks. So it would be nice to know we can pass all of them under stress.

Reply:
This is a parameter, so you can always choose. What we are trying to figure out is the best default for our everyday workflow when running stress tests for some reason.

Reply:
Yes, you can chose, it's just about what's the default. This looks like even split so let's keep the `Release` as default, I don't mind.